WDIO_Tests/Tests/addItemsToCart.js

```diff
@@ -0,0 +1,30 @@
+import { expect, browser, $ } from '@wdio/globals'
+
+describe('Test case perform login to saucedemo and further addition of items to a cart', () => {
+    it( 'Perform login, add items', async () => {
+        await browser.url(`https://www.saucedemo.com/`);
+
+        await $('#user-name').setValue('standard_user');
+        await $('#password').setValue('secret_sauce');
+
+        const loginButton = await $('#login-button');
+        await loginButton.click();
+
+        const bagPack = await $('#item_4_title_link');
+        await expect(bagPack).toBeDisplayed();
+
+        const addToCart = await $('#add-to-cart-sauce-labs-backpack');
```

Comment:
Right now your test will work only for element with selector "#add-to-cart-sauce-labs-backpack" what if that element is not present any more?
Let's change the selector so that test adds the first item.